mw-webapp/src/logic/wayPage/labels/LabelLine.module.scss

```diff
@@ -2,21 +2,40 @@
 
 .labelLine {
   align-items: baseline;
-  gap: $gapBig;
+  gap: 0 $gapBig;
+
+  @media only screen and (max-width: $mediaWidthTablet) {
+    flex-wrap: wrap;
+    padding-bottom: $paddingExtraMedium;
+  }
```

Comment:
why padding only for tablet? why flex wap only for tablet and less

Reply:
We don't require flex-wrap and paddings on the desktop version.

![bug_06_01](https://github.com/user-attachments/assets/019eb4ee-e590-40f6-90d7-483484240817)